src-lib/DarkHelpUtils.cpp

```diff
@@ -118,7 +155,7 @@ DarkHelp::MStr DarkHelp::verify_cfg_and_weights(std::string & cfg_filename, std:
 			else							names_filename		= iter.second;
 		}
 
```

Comment:
Why the whitespace changes in the next few diffs?

Reply:
Fixed.